x-pack/legacy/plugins/infra/index.ts

```diff
@@ -43,7 +43,7 @@ export function infra(kibana: any) {
             defaultMessage: 'Explore your metrics',
           }),
           icon: 'plugins/infra/images/infra_mono_white.svg',
```

Comment:
[nitpicking] should we rename the SVGs as well to keep the naming consistent?

Reply:
I'll leave that decision up to you all; I'm not sure how much infra-to-metrics language conversion is planned for the app. The underlying image actually changed during the switch from `infraApp` to `metricsApp`. Plus, that name is part of the EUI API, so there is some necessity for EUI to align with the direction.